src/services/logger.ts

```diff
@@ -21,21 +22,33 @@ const logFactory: LogFunctionFactory<Logger> = scope => {
       console.log('Fatal error in LogDNA', event);
     });
     if (config.logger.forceLocal) {
-      return (level, message, meta) => {
-        console.log(`${level}: ${message}`, meta);
+      return async (level, message, meta) => {
+        const { dbId } = scope;
+        let userId: string | undefined;
+        if (dbId) {
+          const user = await MetadataRepo.getUserFromDbId(dbId);
```

Comment:
does this happen in the execution of the main path?

Reply:
No, because we never `await` the `logger.` calls, so this occur asynchronously.